eta/core/objects.py

```diff
@@ -107,7 +112,7 @@ def attributes(self):
         _attrs = ["label", "bounding_box"]
         _optional_attrs = [
             "mask", "confidence", "index", "score", "frame_number",
-            "index_in_frame", "eval_type"]
+            "index_in_frame", "eval_type", "event_indices"]
```

Comment:
As implemented, this will always include `event_indices` in every serialized `DetectedObject`. I'd rather only include it if the object has an associated event

Reply:
Right, of course!